examples/ota-provider-app/esp32/main/main.cpp

```diff
@@ -41,8 +41,8 @@
 
 #include <lib/support/ErrorStr.h>
 
-#include <BdxOtaSender.h>
 #include <app/clusters/ota-provider/ota-provider.h>
+#include <ota-provider-common/BdxOtaSender.h>
 #include <ota-provider-common/OTAProviderExample.h>
 
 using chip::BitFlags;
```

Comment:
nit: I think this can be removed

Reply:
Removed this and and other unused `using` lines. `using chip::bdx::TransferControlFlags;` is not required as it was moved to `OTAProviderExample.cpp`